tarsier-service/service/src/auth.py

```diff
@@ -0,0 +1,17 @@
+import os
+from fastapi import Depends, HTTPException, Security
+from fastapi.security import HTTPAuthorizationCredentials, HTTPBearer
+from propelauth_fastapi import init_auth, User
+from dotenv import load_dotenv
+
+
+load_dotenv() 
+auth = init_auth("https://6966894145.propelauthtest.com", os.environ["PROPEL_AUTH_API_KEY"])
```

Comment:
Should place secrets in Pydantic settings for easier management. This is what we do both currently and on AgentGPT https://docs.pydantic.dev/latest/concepts/pydantic_settings/#usage